handbook/queries/README.md

```diff
@@ -30,12 +30,18 @@ Fleet's standard query library includes a growing collection of useful queries f
 - [Get system uptime](./get-system-uptime.md) (macOS, Linux, Windows, FreeBSD)
 - [Get crashes](./get-crashes.md) (macOS)
 
+### Importing the queries in Fleet
```

Comment:
All makes sense.  I think we can get rid of the list of queries above once the markdown files go away.  I think we can run with this YAML format and get it compiling- let's just wait to axe markdown files til we have it all shipped to the website.

Reply:
I concur with what Mike says.

Reply:
Roger roger :battle-droid. Makes sense!